GCP_deploy/main.py

```diff
@@ -0,0 +1,124 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+""" Web app for LaserTagger text summarizer """
+
+from __future__ import print_function
+
+from flask import Flask, render_template, request
+from predict_main import construct_example
+import nltk
+import bert_example
+import utils
+import tagging_converter
+import googleapiclient
+import tagging
+import bert_example_classifier
+from nltk.tokenize.treebank import TreebankWordDetokenizer
+from builtins import FileExistsError
+
+app = Flask(__name__)
+
+embedding_type = "POS"
+label_map_file = "gs://publicly_available_models_yechen/best_hypertuned_POS/label_map.txt"
+enable_masking = False
+do_lower_case = True
+
+try:
+    nltk.download('punkt')
+except FileExistsError:
+    print("NLTK punkt exist")
+
+try:
+    nltk.download('averaged_perceptron_tagger')
+except FileExistsError:
+    print("NLTK averaged_perceptron_tagger exist")
+
+if embedding_type == "Normal" or embedding_type == "Sentence":
+    vocab_file = "gs://lasertagger_training_yechen/cased_L-12_H-768_A-12/vocab.txt"
+elif embedding_type == "POS":
+    vocab_file = "gs://bert_traning_yechen/trained_bert_uncased/bert_POS/vocab.txt"
+elif embedding_type == "POS_concise":
+    vocab_file = "gs://bert_traning_yechen/trained_bert_uncased/bert_POS_concise/vocab.txt"
+else:
+    raise ValueError("Unrecognized embedding type")
+    
+label_map = utils.read_label_map(label_map_file)
+converter = tagging_converter.TaggingConverter(
+    tagging_converter.get_phrase_vocabulary_from_label_map(label_map), True)
+id_2_tag = {tag_id: tagging.Tag(tag) for tag, tag_id in label_map.items()}
+builder = bert_example.BertExampleBuilder(label_map, vocab_file,
+128, do_lower_case, converter, embedding_type, enable_masking)
+
+grammar_vocab_file = "gs://publicly_available_models_yechen/grammar_checker/vocab.txt"
+grammar_builder = bert_example_classifier.BertGrammarExampleBuilder(grammar_vocab_file, 128, False)
```

Comment:
Define them in __init__ function:

https://stackoverflow.com/questions/28925385/python-module-init-function

or define some functions like
_get_embedding_type()

EMBEDDING_TYPE = _get_embbedding_type()